managers/StartBoshManagers.js

```diff
@@ -3,10 +3,12 @@
 const BoshManager = require('./bosh-manager/BoshManager');
 const BoshBindManager = require('./bosh-manager/BoshBindManager');
 const BoshTaskStatusPoller = require('./bosh-manager/BoshTaskStatusPoller');
+const BoshStaggeredDeploymentPoller = require('./bosh-manager/BoshStaggeredDeploymentPoller');
 
 const boshManager = new BoshManager();
 const bindManager = new BoshBindManager();
 /* jshint nonew:false */
 new BoshTaskStatusPoller();
+new BoshStaggeredDeploymentPoller();
```

Comment:
Somehow, I'm not a big fan of having a constructor performing initialization procedures. But, if that's the way we're planning to keep this, it's fine